src/pages/YourShoppingListsHere/LiYuan/index.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components'
+import type { InputRef } from 'antd'
+import type { ColumnType } from 'antd/es/table'
+import { Space, Button, Form, Input, Popconfirm, Table } from 'antd'
+import type { FormInstance } from 'antd/es/form'
+import React, { useContext, useEffect, useRef, useState } from 'react'
+
+// Do not edit the other parts of the code!
+
+const ShoppingListDiv = styled.div`
+  margin: 1.5rem 1rem;
+`
+
+const SL_LiYuan = () => {
+  const data = [
+    {
+      key: 0,
+      name: 'Eggs',
+    },
+    {
+      key: 1,
+      name: 'Pineapples',
+    },
+  ]
+
+  const [Data, setData] = useState(data)
+  const [value, setValue] = useState('')
+
+  const columns = [
+    {
+      key: 'name',
+      title: 'Item Name',
+      dataIndex: 'name',
+    },
+    {
+      key: 'actions',
+      title: 'Actions',
+      width: 400,
+      render: (record: any) => {
+        return (
+          <Button onClick={() => Delete(record)} type="primary" style={{ marginBottom: 16 }}>
+            Delete
+          </Button>
+        )
+      },
+    },
+  ]
+  const [count, setCount] = useState(2)
+
+  const Delete = (record: any) => {
```

Comment:
camelCase (refer to [docs](https://www.notion.so/heads-of-rhdevs/WIP-RHDevs-Frontend-Code-Style-Guide-b5f294c047f14f78b370031afb2c73c5#ebac033cffd84ea5a28cf73f2df30b13) if unsure)
```suggestion
  const delete = (record: any) => {
```